<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QWidget" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>470</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>470</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>331</width>
     <height>331</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>77</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="1" column="1">
     <widget class="QPushButton" name="btn22">
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>77</width>
        <height>54</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="btn62">
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>54</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="btn41">
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>54</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btn23">
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>54</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btn33">
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>54</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="btn42">
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>54</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="btn24">
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>54</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>รท</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btn21">
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>54</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QPushButton" name="btn64">
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>54</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="btn54">
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>54</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QPushButton" name="btn61">
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>54</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>EXIT</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="btn14">
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>54</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="btn44">
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>54</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="btn63">
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>54</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btn32">
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>54</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="btn11">
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>54</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="btn43">
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>54</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="btn34">
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>54</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btn31">
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>54</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="btn53">
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>54</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="btn51">
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>54</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="btn52">
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>54</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btn13">
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>54</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btn12">
      <property name="minimumSize">
       <size>
        <width>77</width>
        <height>54</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>54</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>331</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QLabel" name="fillLab">
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit">
      <property name="minimumSize">
       <size>
        <width>327</width>
        <height>79</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>79</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
